### PR TITLE
Changed Dual Index Plate name

### PR DESCRIPTION
**Pull request by seenanair**
Dual Index Plate names changed to Fresh Frozen - Dual Index TT Set A and
FFPE - Dual Index TS Set A

### CODE REVIEW

**Review by BenTopping — APPROVED** [review]
Looks good. Just wondering if we need a test using 'FFPE - Dual Index TS Set A' plate type. Does the plate type affect any logic?

**Comment by seenanair** [comment]
> Looks good. Just wondering if we need a test using 'FFPE - Dual Index TS Set A' plate type. Does the plate type affect any logic?

Not actually on client side. It is just another field to pass on to the mutation.

**Comment by seenanair** [comment]
@BenTopping  But another thought, while you are here, I will give it a go with unit test cases for this story

File: tests/unit/components/slotMapper/reagentTransferSlotMapper.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+const destLabware = createLabware('STAN-5111');
+
+describe('ReagentTransferSlotMapper.tsx', () => {
+  it('displays the parent div elements for source and destination', () => {
+    const reagentSlotMapper = render(
+      <ReagentTransferSlotMapper initialSourceLabware={undefined} initialDestLabware={undefined} />
+    );
+
+    //Displays div element for source labware
+    const sourceLabwareElem = getById(reagentSlotMapper.container, 'sourceLabwares');
```
Comment (BenTopping) [inline tests/unit/components/slotMapper/reagentTransferSlotMapper.spec.tsx:21]: You could add a `data-testid=sourceLabwares` to the reagentTransferSlotMapper components to use as a unique identifier for the tests. Then get the component by its testId in the tests like c`onst sourceLabwareElem = screen.getByTestId('sourceLabwares')`

File: tests/unit/components/slotMapper/reagentTransferSlotMapper.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+enableMapSet();
+afterEach(() => {
+  cleanup();
+});
+const srcLabware = createLabware('123456789123456789012345');
+const destLabware = createLabware('STAN-5111');
+
+describe('ReagentTransferSlotMapper.tsx', () => {
+  it('displays the parent div elements for source and destination', () => {
+    const reagentSlotMapper = render(
```
Comment (BenTopping) [inline tests/unit/components/slotMapper/reagentTransferSlotMapper.spec.tsx:16]: not sure assigning it a variable is necessary as you can reference the rendered element through`screen`

**Pull request merged**